app/models/concerns/exportable.rb

```diff
@@ -0,0 +1,15 @@
+module Exportable
+  extend ActiveSupport::Concern
```

Comment:
Thoughts on whether to just make this a plain old Module and mix it in like plain old Ruby?